rir/src/interpreter/interp.cpp

```diff
@@ -3435,7 +3435,9 @@ SEXP evalRirCode(Code* c, InterpreterInstance* ctx, SEXP env,
         }
 
         INSTRUCTION(guard_fun_) {
+#ifndef UNSOUND_OPTS
```

Comment:
why is this here? an env flag and a define flag with exactly the same name is very confusing. also, with the other change this is probably not needed anymore.